code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -41,6 +41,23 @@
 	gold_core_spawnable = 0
 
 /mob/living/simple_animal/pet/cat/Life()
+	if(!stat && !buckled)
+		if(prob(1))
+			emote("me", 1, pick("stretches out for a belly rub.", "wags its tail."))

```

Comment:
They can still bite, unless they were demouthed...

Why not "wags \her tail" /  "wags \his tail"? You put gender on grooming.